packages/core/src/WebPlayer.ts

```diff
@@ -153,7 +153,7 @@ export default class WebPlayer extends EventEmitter {
       this.tech = null;
       this.emit(PlayerEvent.UNREADY);
 
-      this.emit(PlayerEvent.DESTROY_EPAS);
+      this.emit(PlayerEvent.DEINIT_EPAS);
```

Comment:
Why even have a specific event and not just use the `UNREADY` event above?

Reply:
Removed event; changed to listening to `UNREADY`